src/test/java/seedu/guest/testutil/EditGuestDescriptorBuilder.java

```diff
@@ -16,22 +16,22 @@
 /**
  * A utility class to help with building EditPersonDescriptor objects.
  */
-public class EditPersonDescriptorBuilder {
+public class EditGuestDescriptorBuilder {
 
     private EditGuestDescriptor descriptor;
 
-    public EditPersonDescriptorBuilder() {
+    public EditGuestDescriptorBuilder() {
         descriptor = new EditGuestDescriptor();
     }
 
-    public EditPersonDescriptorBuilder(EditGuestDescriptor descriptor) {
+    public EditGuestDescriptorBuilder(EditGuestDescriptor descriptor) {
         this.descriptor = new EditGuestDescriptor(descriptor);
     }
 
     /**
      * Returns an {@code EditPersonDescriptor} with fields containing {@code person}'s details
```

Comment:
Should this be EditGuestDescriptor instead of EditPersonDescriptor? And @code guest instead of @code person?